pl_examples/utils/loaders.py

```diff
@@ -0,0 +1,101 @@
+from threading import Thread
+from queue import Queue
+
+import torch
+from torch import Tensor
+import torch.nn as nn
+import torch.nn.functional as F
```

Comment:
It seems some of these imports are unused

Reply:
Yep, will deal with those

Reply:
@awaelchli Done! Although I think it's failing the CI build and tests because I resolved the conflict with a merge from master and master is currently failing